repos/fdbt-site/src/pages/api/chooseValidity.ts

```diff
@@ -0,0 +1,65 @@
+import { NextApiResponse } from 'next';
+import { NextApiRequestWithSession, ErrorInfo } from '../../interfaces/index';
+import { updateSessionAttribute } from '../../utils/sessions';
+import { DURATION_VALID_ATTRIBUTE } from '../../constants/attributes';
+import { redirectToError, redirectTo } from './apiUtils';
+import { isValidInputDuration } from './apiUtils/validator';
+
+export const isInvalidValidityNumber = (validityInput: number): boolean => {
```

Comment:
Not sure about the word validity. Maybe just `isValidNumber`? And then `input` for the variable?